iquip/apps/scheduler.py

```diff
@@ -0,0 +1,336 @@
+"""App module for showing the experiment list."""
+
+from typing import Optional, Tuple, List, Literal
+
+from PyQt5.QtGui import QPainter
+from PyQt5.QtCore import Qt, QObject, QAbstractListModel, QModelIndex, QMimeData, QSize
+from PyQt5.QtWidgets import (
+    QStyleOptionViewItem, QWidget, QLayout, QLabel, QListView,
+    QPushButton, QHBoxLayout, QVBoxLayout, QAbstractItemDelegate, QAction
+)
+
+import qiwis
+from iquip.protocols import ExperimentInfo
+
+def _dismiss_items(layout: Optional[QLayout] = None):
+    """Decouples components in the layout.
+
+    Args:
+        layout: The layout whose elements are intended to be deleted.
+    """
+    if layout is not None:
+        while layout.count():
+            item = layout.takeAt(0)
+            widget = item.widget()
+            if widget is not None:
+                widget.setParent(None)
+                layout.removeWidget(widget)
+                widget.deleteLater()
+                widget = None
+            else:
+                _dismiss_items(item.layout())
+
+
+class SchedulerFrame(QWidget):
+    """Frame for displaying the submitted experiment list.
+    
+    Attributes:
+        expRun: The ExperimentView widget displaying currently running experiment.
+        expList: The QListView widget holding queued experiments.
+        model: The ExperimentModel that manages the data of expList.
+    """
+
+    def __init__(self, parent: Optional[QWidget] = None):
+        """Extended."""
+        super().__init__(parent=parent)
+        # widgets
+        self.expRun = RunningExperimentView()
+        self.expList = QListView()
```

Comment:
I'm sorry to point this out now, I had no time to read it thoroughly.
We name widgets using their types, e.g., ~Button, ~Label, ~Box, etc.
This helps readers notice the widget type at a glance.
Furthermore, it clearly shows that the variable is a widget.

In this case, I would suggest `runningView` and `queueView`. Of course these are just examples. Please name them with clearly readable ones.